build

```diff
@@ -7,20 +7,26 @@ try {
 	process.exit(1);
 }
 
-var child_process = require('child_process');
-var fs = require('fs');
-var path = require('path');
+const child_process = require('child_process');
+const fs = require('fs');
+const path = require('path');
```

Comment:
I've been using `var` in build scripts just in case ridiculously old versions of Node don't support them, but I guess that's a really obscure worry.

Reply:
Oh, didn't realize. I thought we were safe because of the version check. Reverted - no point in breaking things if we don't have to.